oidc-controller/api/core/oidc/provider.py

```diff
@@ -1,26 +1,56 @@
 import logging
+import os
 
+from api.core.config import settings
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
 from pymongo.database import Database
-from urllib.parse import urlparse
-from jwkest.jwk import rsa_load, RSAKey, KEYS
-
 from pyop.authz_state import AuthorizationState
 from pyop.provider import Provider
 from pyop.subject_identifier import HashBasedSubjectIdentifierFactory
 from pyop.userinfo import Userinfo
+from urllib.parse import urlparse
+from jwkest.jwk import rsa_load, RSAKey, KEYS
 
+logger = logging.getLogger(__name__)
+FILE_PATH = os.path.dirname(os.path.realpath(__file__)).replace("/api/core/oidc", "")
```

Comment:
Let's just provide a reasonable default: I think a default path that is external to the code folder structure is best, could be something like `/app/signing-keys` (I think the Dockerfile has all the application code in the `/app` folder).
Make sure to handle the scenario where the folder does not exist - you'll want to build/run the project in docker to confirm things are working as expected.

c.c.: @WadeBarnes for additional input, if required

Reply:
For unit test, it will create `test_signing_key.pem` in `signing-keys` directory at the same level as `oidc-controller` directory.

With docker [manage script], it will create `/app/signing-keys` directory.